### DISCUSSION TITLE
发评论提示Failed to fetch

### DISCUSSION THREAD

**Discussion by OrdChaos** [walinejs/waline#950] [Q&A] [answered]
在博客上发评论，点击发送按钮后会加载很久，然后提示Failed to fetch，不过可以发出评论。
感觉问题可能不出在waline，不过我也不确定。因为网站上我同时使用了freecdnjs，而发评论在控制台中的报错则是
![image](https://user-images.githubusercontent.com/101442840/164580046-e9ae0803-7b79-4578-9eb7-0faa87b5c7ac.png)
求教

**Comment by lizheming** [comment, marked as answer]
你看下是不是配了邮箱通知之类的，可以去掉一下看看，我看是 Vercel 执行超时导致的报错。
